xud.conf

```diff
@@ -0,0 +1,21 @@
+lndDir = "~/.lnd"
+rpcPort = 8886
+
+[db]
+username = "xud"
+password = ""
+database = "xud"
+port = 3306
+host = "db"
+
+[p2p]
+listen = true
+port = 8885
+
+[lnd]
+disable = false
+rpcprotopath = "lndrpc.proto"
+
+[raiden]
+disable = false
+port = 5001
```

Comment:
It looks like these are all the default values, right? If so, we shouldn't need to explicitly create or define a config file for the docker container, default values are hardcoded and loaded automatically.

Reply:
We need to be able to inject the MySQL and other containers connection details. hence, I think this is required.

Reply:
I'm not sure what you mean, is this file being used by something other than xud? The connection details specified here are hardcoded into xud.

Reply:
This file is used by the `xud` container if you could refer to docker-compose.yml for connecting to MySQL.